packages/frontend/src/components/SlideCard.tsx

```diff
@@ -0,0 +1,80 @@
+import classNames from 'classnames'
+import React, { ReactNode } from 'react'
+
+interface SlideCardProps {
+  className?: string
+  title: string
+  button: ReactNode
+  closeButtonText?: string
+  children: ReactNode
+}
+
+export function SlideCard(props: SlideCardProps) {
+  return (
+    <div
+      className={classNames(
+        'SlideCard group relative w-min whitespace-pre',
+        props.className,
+      )}
+    >
+      <SlideCardToggle type="open">{props.button}</SlideCardToggle>
+      <SlideCardContent
+        title={props.title}
+        closeButtonText={props.closeButtonText}
+      >
+        {props.children}
+      </SlideCardContent>
+    </div>
+  )
+}
+
+interface SlideCardToggleProps {
+  children: ReactNode
+  type: 'open' | 'close'
+}
+
+function SlideCardToggle(props: SlideCardToggleProps) {
+  return (
+    <label
+      className="flex cursor-pointer items-center justify-between"
+      data-role="chart-token-toggle"
+    >
+      <input
+        type="checkbox"
+        autoComplete="off"
+        className="SlideCard-Toggle peer hidden"
+        data-role={props.type}
+      />
+      {props.children}
+    </label>
+  )
+}
+
+interface SlideCardContentProps {
+  title: string
+  closeButtonText?: string
+  children: ReactNode
+}
+
+function SlideCardContent(props: SlideCardContentProps) {
+  return (
+    <div className="SlideCard-Content transition-300 fixed left-0 top-0 bottom-0 z-999 flex h-[100dvh] w-full translate-y-full flex-col whitespace-normal bg-black/50 transition-transform ease-out">
+      <div className="h-[10%]" />
+      <div className="flex h-[90%] flex-col rounded-t-2xl bg-gray-100 dark:bg-zinc-800">
+        <h2 className="m-4 mb-0 text-3xl font-bold">{props.title}</h2>
```

Comment:
This guy's beefy, I feel like maybe we could get rid of that title? Wdyt?

![image](https://github.com/l2beat/l2beat/assets/67391475/3bdc2d3c-0aa6-4c0b-9e6c-2dd684e3d117)

Reply:
I don't really mind it, otherwise the visual noise it too great I think with all the different coin logos and also without the list just kinda starts weirdly